tests/rendered_output.rs

```diff
@@ -28,62 +32,76 @@ fn by_default_mdbook_generates_rendered_content_in_the_book_directory() {
 
 #[test]
 fn make_sure_bottom_level_files_contain_links_to_chapters() {
-    let temp = helpers::DummyBook::default().build();
+    let temp = DummyBook::default().build();
     let mut md = MDBook::new(temp.path());
     md.build().unwrap();
 
     let dest = temp.path().join("book");
     let links = vec![
-        "intro.html",
-        "first/index.html",
-        "first/nested.html",
-        "second.html",
-        "conclusion.html",
+        r#"href="intro.html""#,
```

Comment:
Nice fix! Without the `href="` this test wouldn't have actually been checking for links, just that the filenames are present, which means it's not actually testing what we want it to.